v21.1/spatial-tutorial.md

```diff
@@ -0,0 +1,1835 @@
+---
+title: Spatial Tutorial
+summary: Tutorial for working with spatial data in CockroachDB.
+toc: true
```

Comment:
I'll find time today and tomorrow for a thorough review, @rmloveland, but in the meantime, please add `toc_not_nested: true` as a fourth front-matter field. Right now, the h3s in the instructions are exploding the TOC and making it hard to get a sense of the whole at a glance:

<img width="552" alt="Screen Shot 2021-02-02 at 7 43 00 AM" src="https://user-images.githubusercontent.com/15893490/106601950-4bd1bb00-652a-11eb-9156-01fddfb164aa.png">

Reply:
Just pushed the update!